db/migrate/20231020151031_drop_present_at_unpublishing_column_from_attachments.rb

```diff
@@ -0,0 +1,5 @@
+class DropPresentAtUnpublishingColumnFromAttachments < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :attachment_data, :present_at_unpublish, :boolean
```

Comment:
If you deploy this out for testing you might run into an issue on merge where the build fails due to the column already having been dropped (unless you wait for the db to sync overnight).

If you want to merge today it might be worth adding if_exists

```
remove_column :attachment_data, :present_at_unpublish, :boolean, if_exists: true
```